### ISSUE TITLE
Some select inputs field not closing on click 

### ISSUE THREAD

**Issue by anikdhabal** [RocketChat/Rocket.Chat#28936]
### Description:

Basically, chevron down Icon is used for showing a dropdown list & chevron up Icon is used for hiding that dropdown list after clicking. But in our case only the Chevron down button is working and unable to hide the dropdown list using it. It's working fine in 
many inputs field and also not working in others.

### Expected behavior:

https://user-images.githubusercontent.com/81948346/232480288-ea852082-54c3-4a2c-a670-ee765ec05cd1.mp4

### Actual behavior:

https://user-images.githubusercontent.com/81948346/232482505-a5efa7f0-eae2-44f6-80b2-7280ac6ac445.mp4

### Server Setup Information:
-Version of Rocket.Chat Server: 6.0.0-develop
-Operating System: windows 10
-Deployment Method: Gitpod
-Number of Running Instances:
-DB Replicaset Oplog: Enabled
-NodeJS Version: 14.21.2 - x64
-MongoDB Version:5.0.5

**Comment by thealoneshadow** [comment]
Can i work on this one?

**Comment by hugocostadev** [comment]
The problem it's not with the chevron-up icon... If you notice, some dropdown inputs do not close on clicking, just if you click outside... 

Could be flagged as an improvement, but for certain inputs, I think that makes sense this behavior, what do you think?

**Comment by anikdhabal** [comment]
@hugocostadev Yes sir this Problem occurs due to the inputs. Thanks for your response.
